Google.PowerShell/Common/CloudSdkSettings.cs

```diff
@@ -21,6 +21,18 @@ public class CommonProperties
             public const string Region = "region";
         }
 
+        /// <summary>
+        /// Environment variable which points to the location of the current configuration file.
+        /// This overrides the user configuration file as well as the globa lcinstallation properties file.
+        /// </summary>
+        private const string CloudSdkConfigVariable = "CLOUDSDK_CONFIG";
+
+        /// <summary>
+        /// Environment variable which stores the current active config.
+        /// This overrides value found in active_config file.
```

Comment:
Environment variable which stores the current active config. Overrides the value found in the active_config file *if present*.